scripts/src/java/org/oppia/android/scripts/regex/RegexPatternValidationCheck.kt

```diff
@@ -226,9 +246,17 @@ private data class MatchableFileContentCheck(
    * provided iterable.
    */
   fun computeAffectedLines(lines: Iterable<String>): List<Int> {
-    return lines.withIndex().filter { (_, line) ->
-      prohibitedContentRegex.containsMatchIn(line)
-    }.map { (index, _) -> index }
+    if (prohibitedContentRegex != null)
+      return lines.withIndex().filter { (_, line) ->
+        prohibitedContentRegex.containsMatchIn(line)
+      }.map { (index, _) -> index }
+    else return emptyList()
+  }
+
+  /** Returns a boolean indicating whether the file contains the required content or not. */
+  fun doesFileContainRequiredRegex(fileContent: String): Boolean {
+    if (requiredContentRegex != null) return requiredContentRegex.containsMatchIn(fileContent)
+    else return true
```

Comment:
```suggestion
    requiredContentRegex?.containsMatchIn(fileContent) ?: true
```

More concise & idiomatic.

Reply:
done.